src/polyfills/ElementPolyfill.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+if (!Element.prototype.matches) {
```

Comment:
I'm ok with this just being written out, but there should at least be a link saying where this code came from... i.e. https://github.com/idmadj/element-closest-polyfill/blob/master/index.js

Reply:
@travist I got that from https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Reply:
I meant in the actual code. Like in the comments have that link so when we run across it we know where it came from.